glotaran/builtin/io/pandas/test/test_pandas_parameters.py

```diff
@@ -0,0 +1,97 @@
+from __future__ import annotations
+
+from pathlib import Path
+
+from glotaran.io import load_parameters
+from glotaran.io import save_parameters
+
+PANDAS_TEST_DATA = Path(__file__).parent / "data"
+PATH_XLSX = PANDAS_TEST_DATA / "parameter.xlsx"
+PATH_ODS = PANDAS_TEST_DATA / "parameter.ods"
+PATH_CSV = PANDAS_TEST_DATA / "parameter.csv"
+PATH_TSV = PANDAS_TEST_DATA / "parameter.tsv"
+PATH_YAML = PANDAS_TEST_DATA / "parameter.yaml"
+
+
+def test_load_parameters_xlsx():
+    """load parameters file as yaml and excel and compare them"""
+    parameters_xlsx = load_parameters(PATH_XLSX)
+    parameters_yaml = load_parameters(PATH_YAML)
+    assert parameters_yaml == parameters_xlsx
+
+
+def test_save_parameters_xlsx(
+    tmp_path: Path,
+):
+    """load parameters file from yaml and save as xlsx
+    and compare with yaml and reloaded xlsx parameter files"""
+    parameters_yaml = load_parameters(PATH_YAML)
+    parameter_path = tmp_path / "testparameters.xlsx"
+    save_parameters(file_name=parameter_path, format_name="xlsx", parameters=parameters_yaml)
```

Comment:
Somewhat counter intuitive, you could change the format_name to `format_name="ods"` and it would still  work (unit tests still pass), because it would select the plugin based on format name, but because that's the same plugin as used for 'xlsx' saving the file still works. Under the hood pandas.to_excel is used which by itself infers how to save it based on the file extension, the format_name information is not propagated. 

Not something I expect to be an issue, but important to note.